spec/dummy/db/migrate/20201125122162_add_indices_and_visibility_fields.spina_blog.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+# This migration comes from spina_blog (originally 7)
+
+class AddIndicesAndVisibilityFields < ActiveRecord::Migration[5.2]
+  def change
+    change_table :spina_blog_posts, bulk: true do |t|
+      t.boolean :corp, default: false, null: false, index: true
+      t.boolean :back_office, default: false, null: false, index: true
```

Comment:
I would add enum field - for example "scope"/"namespace" which could be configured in config.
```
Spina::Blog.configure do |config|
  config.scopes = [:corp, :backoffice, :global]
end

# app/models/spina/post.rb
enum scope: Spina::Blog.config.scopes
```

so this library had an isolation from app. maybe it could be reused in other WS apps

Reply:
I added separate checkboxes as @tonic20 suggested, which came as more intuitive from a user perspective too. I guess we can use enum under the hood but show checkboxes, but probably with more complexity.

@tonic20 if you also think that an enum/dropdown is better, I can modify it.

Reply:
Well yes I think checkboxes better idea from UI perspective. Let's keep it as is and refactor it later if we will need to customize post scope.